lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -208,6 +208,8 @@ void LLVMToSPIRVDbgTran::transLocationInfo() {
           // According to the spec, OpLine for an OpBranch/OpBranchConditional
           // must precede the merge instruction and not the branch instruction
           auto *VPrev = static_cast<SPIRVInstruction *>(V)->getPrevious();
+          if (!VPrev)
```

Comment:
Even if `VPref` is null, we still need to call `BM->addLine()`

Reply:
True. At first I cringed that regression tests didn't catch this.

Reply:
Done